RecoEgamma/EgammaTools/interface/PhotonIDHelper.h

```diff
@@ -0,0 +1,86 @@
+//--------------------------------------------------------------------------------------------------
+//
+// EGammaID Helper
+//
+// Helper Class to compute photon ID observables
+//
+// Authors: F. Beaudette,A. Lobanov
+//--------------------------------------------------------------------------------------------------
+
+#ifndef PhotonIDHelper_H
+#define PhotonIDHelper_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h